packages/jaeger-ui/src/components/Monitor/ServicesView/index.css

```diff
@@ -33,8 +33,6 @@ limitations under the License.
 .timeframe-selector {
   display: inline-flex;
   justify-content: flex-end;
-  position: absolute;
-  bottom: 14px;
```

Comment:
The position of the Timeframe Selector needs to be relative to the Col beside it.

Due to `position: absolute`, the Selector was going to the bottom left corner, which is not the desired behavior.

`bottom: 14px` is removed because it is already at an appropriate distance from the col below it.

In Antd 3x:
![](https://user-images.githubusercontent.com/94157520/263591344-27c9cc08-c864-4eae-8c0d-99dab8d6069d.PNG)

With this PR:
![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/858060e7-a1ee-4ba5-9e5e-eb9dc05e3ad7)